### DISCUSSION TITLE
How do I access WG_ENABLE_EXPIRES_TIME, UI_ENABLE_SORT_CLIENTS and WG_ENABLE_ONE_TIME_LINKS on Web UI?

### DISCUSSION THREAD

**Discussion by afunworm** [wg-easy/wg-easy#1315] [Q&A] [answered]
I've updated wg-easy to the latest version (:latest) and enable WG_ENABLE_EXPIRES_TIME, UI_ENABLE_SORT_CLIENTS and WG_ENABLE_ONE_TIME_LINKS. However, in the UI, I don't see anything that makes use of these variables.

Am I missing something?

**Comment by sashaj** [comment]
Having similar issue on the latest version.
I'm somewhat confused, how do i insert boolean values?
is it, for example:
-e UI_ENABLE_SORT_CLIENTS=true
or
-e UI_ENABLE_SORT_CLIENTS="true"
or
-e UI_ENABLE_SORT_CLIENTS='true' \

I've tried:
=TRUE
=true
="TRUE"
='true'
="true"

but nothing seems to work, no changes in web UI, /metrics always returns 404

**Comment by eandersons** [comment, marked as answer]
See #1325.
